dumper/app/build.gradle

```diff
@@ -60,14 +60,14 @@ dependencies {
     implementation libs.google.cloud.nio
     implementation libs.google.cloud.kms
     implementation libs.httpclient5
+    implementation libs.aws.java.sdk.redshift
```

Comment:
It's because dependency declared with `implementation` is available in compile time and runtime. We now use this library in the dumper code so this needs to be available in the compile time.